components/plugin_marketplace/marketplace_item/marketplace_item.jsx

```diff
@@ -87,7 +87,7 @@ export default class MarketplaceItem extends React.Component {
             this.setState({installing: true});
             trackEvent('plugins', 'ui_marketplace_download');
```

Comment:
shouldn't we remove this? We are tracking in the redux, right?

Reply:
Yes, as @ali-farooq0 suggested, we can know track it at the time of the api call given that we have the dedicated one for marketplace install.

Reply:
@marianunez 
I'm personally OK keeping both, because once 5.16 is released, our analytics tool will contain this event. However >= 5.18 it'll be the other redux event, it might be hard to put together that data if you need to see how many people installed a marketplace plugin. My suggestion would to keep both now. If this change was going into 5.16 then it would've been fine removing that event.

Reply:
Guys, should we just merge this PR and I can follow up with Aaron later to fix the tracking events. 
Follow up discussion: https://github.com/mattermost/mattermost-redux/pull/938#discussion_r333000012

Created a jira to keep track: https://mattermost.atlassian.net/browse/MM-19318